pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [tool.poetry]
-name = "sparklml-base-classes"
+name = "sparklml_base_classes"
```

Comment:
```suggestion
name = "sparkml-base-classes"
```

Reply:
As I understand, the following is the file structure you are proposing,

```
project
│   README.md
│   poetry.loc
│   pyproject.toml 
│
└───src/
│   └───sparkml_base_classes
|       |   __init__.py
│       │   sparkml_base_classes.py
│   
└───tests/
    │   __init__.py
    │   sparkml_base_classes_test.py
```

However, I believe that once installed, in order to achieve something like `from sparkml_base_classes import EstimatorBaseClass, TransformerBaseClass`, the :mod:`sparkml_base_classes.__init__.py` needs to be nonempty and with the following content


```python
from .sparkml_base_classes import EstimatorBaseClass, TransformerBaseClass
```

Regarding the project renaming from sparklml-base-classes -> sparklml_base_classes, I thought it was necessary for the installation process since it will look after `src/<package-name>`. Or this is not the case?

Reply:
Yes correct, you can leave the contents in the `__init__.py` file.
Poetry will handle the changes of `-` to `_`. Is somewhat convention that packages names use `-` which when importing and such will use `_`.